### PR TITLE
fix: override options for hybrid deployments

### PR DESCRIPTION
**Pull request by bvs-langchain**
* add new `clickhouse.external.hybrid` option: if `true`, send all payloads to blob storage and disable search

testing: ran a `helm upgrade ... --dry-run` with both `hybrid` values and observed expected env vars

### CODE REVIEW

**Review by langchain-infra — COMMENTED** [review]
one cmt

File: charts/langsmith/examples/blob_storage_config.yaml [hunk 1]
```diff
@@ -3,14 +3,25 @@ config:
   apiKeySalt: "YOUR_API_KEY_SALT"
   blobStorage:
     enabled: true
+    # Set engine: "Azure" if using Azure blob storage
+    engine: "S3"
+    # Set hosting: "hybrid" if using managed ClickHouse
+    hosting: "self-hosted"
```
[the remark this answers is not in the document]
Comment (bvs-langchain) [reply charts/langsmith/examples/blob_storage_config.yaml]: that makes sense to me, updated

File: charts/langsmith/values.yaml [hunk 2]
```diff
@@ -388,6 +390,8 @@ clickhouse:
     # If enabled, use the following values to connect to an external database. This will also disable the
     # creation of a clickhouse stateful-set and service.
     enabled: false
+    # -- Either "self-hosted" or "hybrid"
+    hosting: "self-hosted"
```
Comment (langchain-infra) [inline charts/langsmith/values.yaml:394]: why not just boolean here? hybrid: true or false?

**Review by langchain-infra — APPROVED** [review]
nit but why not just use a boolean?

File: charts/langsmith/values.yaml (hunk 2 above)
Comment (bvs-langchain) [reply charts/langsmith/values.yaml:394]: had been thinking we would set to `cloud` and potentially add as env var, but yeah I don't think that's necessary. will update

**Pull request merged**